### ISSUE TITLE
Eliminate copied code in visualization

### ISSUE THREAD

**Issue by bplimley** [bearing/radwatch-analysis#13]
Specifically, the left/right/peak channel finding from `Gamma_Analysis.py`.

A code that takes a spectrum and an energy as input, and returns channels for left/right/peak, would solve the problem, and probably be useful elsewhere too.

**Comment by bplimley** [comment]
This should be done once Mustapha's branch is merged into `working-branch` or `master` and `Gamma_Analysis.py` uses `ROI_Maker.py` properly.

**Comment by bplimley** [comment]
Closed by #38
